Makefile

```diff
@@ -85,3 +85,11 @@ docker-internal-linkcheck:
 container-internal-linkcheck: link-checker-image-pull
 	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hugo --config config.toml,linkcheck-config.toml --buildFuture
 	$(CONTAINER_ENGINE) run --mount type=bind,source=$(CURDIR),target=/test --rm wjdp/htmltest htmltest
+
+clean-api-reference:
+	rm -rf content/en/docs/reference/kubernetes-api/ref/
```

Comment:
You need to remove all the files every time the reference pages are built?

Reply:
Yes, as the tool will rebuild all the necessary pages. 

This way, we are sure there are no old pages left in the directory after a new processing.